collector/gunicorn/docker-compose.yaml

```diff
@@ -0,0 +1,21 @@
+version: "3.9"
+
+services:
+  otel-collector:
+    image: otel/opentelemetry-collector-contrib:0.56.0
```

Comment:
I think the better approach for the example merge will be to increase to current version and just recognize that right now that will not log as well for us as 0.56. Looking forward to what you find about that issue.